app/components/newsletter-signup/template.hbs

```diff
@@ -1,37 +1,20 @@
 <div class="l-container l-container--touch-full">
   <div class="newsletter-signup__inner">
-    <img src="/assets/img/wqxr__newsletter-signup.png"
-         srcset="/assets/img/wqxr__newsletter-signup@2x.png 2x,
-                 /assets/img/wqxr__newsletter-signup@3x.png 3x"
+    <img src="https://media.wnyc.org/i/raw/2021/05/Beethoven_Final.png"
+         srcset="https://media.wnyc.org/i/raw/2021/05/Beethoven_Final.png 2x,
+                 https://media.wnyc.org/i/raw/2021/05/Beethoven_Final.png 3x"
```

Comment:
If possible, use relative paths here (eg `/i/raw/2021/05/Beethoven_Final.png` instead of `https://media.wnyc.org/i/raw/2021/05/Beethoven_Final.png`